doc/hooks/versionCheckHook.section.md

```diff
@@ -22,14 +24,32 @@ stdenv.mkDerivation (finalAttrs: {
 })
 ```
 
-Note that for [`buildPythonPackage`](#buildpythonpackage-function) and [`buildPythonApplication`](#buildpythonapplication-function), `doInstallCheck` is enabled by default.
+Note that some builders, e.g. [`buildPythonPackage`](#buildpythonpackage-function) and [`buildPythonApplication`](#buildpythonapplication-function), enable `doInstallCheck` by default.
```

Comment:
It's not such a big effort IMO to maintain here a list of all builders that enable `installCheck` phase by default. Hence, the previous phrasing is a bit more decisive and informative. This phrasing sort of suggests: "We don't keep track of which builders enable `installCheck` by default and which do, so you should inspect the source code of each if you don't want to enable it when not needed."

Reply:
Hum, should this paragraph be rewritten each time a new custom builder appears? I don't think so.
This paragraph looks like an example of builder, not a exhaustive listing.

Further, the documentation of these builders should provide this information, and it is the duty of a contributor to read the documentation of that builder at least.
If this info is not found in the docs, a bug report should be opened and the docs team should be pinged.

Reply:
> Further, the documentation of these builders should provide this information, and it is the duty of a contributor to read the documentation of that builder at least.
> If this info is not found in the docs, a bug report should be opened and the docs team should be pinged.

I agree with that. But as for:

> Hum, should this paragraph be rewritten each time a new custom builder appears? I don't think so.
> This paragraph looks like an example of builder, not a exhaustive listing.

Nowadays, the list of builders that enable `installCheck` is very small, so I don't find it exhaustive to list them. I agree that theoretically if a new builder that enables `installCheck` will be added in the future to Nixpkgs, probably no one will remember or know to add it to this "list", but I doubt that will happen TBH, from my experience with Nixpkgs.

The previous phrasing also didn't suggest that the builders mentioned is the definitive list of builders that enable `installCheck` by default. I just say that your phrasing slightly more implies that maybe many more builders are enabling `installCheck`, and that is not true.

Reply:
Reverted, in terms. I reworded the example in a more direct sentence.